test/graphql/queries/benchmark_query_test.rb

```diff
@@ -5,8 +5,8 @@
 class BenchmarkQueryTest < ActiveSupport::TestCase
   test 'query benchmark owned by the user' do
     query = <<-GRAPHQL
-      query allBenchmarks {
-          allBenchmarks {
+      query latestBenchmarks {
+          latestBenchmarks {
```

Comment:
Fine for this PR, but this has me thinking that we should match the TestResult API and allow `bool latest` as an argument.

Reply:
Agreed - ideally all fields can be defined as `collection_field :systems, Types::System` - and just parameterized via the "search" field